resources/winsetup/Yarn.wxs

```diff
@@ -36,6 +36,15 @@
 					Action="set"
 					System="yes"
 				/>
+				<Environment
+					Id="BinPath"
+					Name="PATH"
+					Value="[LocalAppDataFolder]Yarn\.bin"
+					Permanent="no"
+					Part="last"
+					Action="set"
+					System="yes"
```

Comment:
Good catch, thanks @Kovensky! I updated it to add it to the user environment variable rather than the system one. That should work fine. I don't think the shell expands environment variables like `%LocalAppData%` when resolving the PATH.